troposphere/ecs.py

```diff
@@ -1,5 +1,5 @@
 from . import AWSObject, AWSProperty
-from .validators import boolean, network_port, positive_integer
+from .validators import boolean, network_port, positive_integer, integer
```

Comment:
Small change: can you alphabetize the imports? (put integer between boolean and network_port)